mypy/stubgenc.py

```diff
@@ -237,6 +239,26 @@ def __init__(
         self.resort_members = self.is_c_module
         super().__init__(_all_, include_private, export_less, include_docstrings)
         self.module_name = module_name
+        if self.is_c_module:
+            # Add additional implicit imports.
+            # C-extensions are given more lattitude since they do not import the typing module.
```

Comment:
Any reason for this particular choice of names? Obvious omissions are Sequence and Set.

Reply:
I restored what was there before.  I was thinking of inspecting the contents of `typing` and `typing_extensions` modules to get the full list.  Adding more names increases the likelihood of a name clash with first party code (the reason I removed these in the first place), so first I'd like to add a new flag,`--no-implicit-typing-import`, to disable this functionality for users who don't want it.  Give me a few days to find some time to do that.

Reply:
Ok, I'm going to make the call and say that I don't have additional time to invest in this right now, I'm sorry.

 In its current state this PR restores the previous behavior and fixes all known regressions, so I think this I think we should merge this soon as-is, and consider cherry-picking it to a 1.7.x release, if there are any plans for that.   I'd like to get these problems fixed before more people are affected by them.

I really want to add a `--no-implicit-typing-import` flag to `stubgen` but it can wait for another PR.